presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkPhysicalResourceCalculator.java

```diff
@@ -63,7 +64,7 @@ public PhysicalResourceSettings calculate(PlanNode plan, PrestoSparkSourceStatsC
     {
         int hashPartitionCount = getHashPartitionCount(session);
         OptionalInt maxExecutorCount = OptionalInt.empty();
-        PhysicalResourceSettings defaultResourceSettings = new PhysicalResourceSettings(hashPartitionCount, maxExecutorCount);
+        PhysicalResourceSettings defaultResourceSettings = new PhysicalResourceSettings(hashPartitionCount, maxExecutorCount, ImmutableSet.of());
```

Comment:
Do you want to call this extraClientTags ?

Reply:
Are you suggesting that we should rename `physicalResourceOptimizationTags` to `extraClientTags` or asking if we can use a better name instead of `extraClientTags` inside QueryInfo? Sorry, bit confused about the comment.

Reply:
Can we call it something like - `runtimeTags`, `serverTags`. I believe the naming of `clientTags` represents tags passed by clients ?